tests/variable_creation_test.py

```diff
@@ -475,6 +475,35 @@ def test_arange():
     assert sc.identical(var, expected)
 
 
+def test_arange_datetime_from_int():
+    var = sc.arange('t', 14, 65, 21, unit='s', dtype='datetime64')
+    expected = sc.datetimes(dims=['t'], values=[14, 35, 56], unit='s')
```

Comment:
I would *not* expect this. I feel this is a bad API, since it hides the epoch.

Reply:
Do you mean for `arange`? All functions creating datetimes can do this (`array`, `scalar`, `datetime[s]`)

Reply:
I guess I mean all. But I suppose this is the only way that avoid extra operations of you have a time since epoch as input?

Reply:
Yes. If we want to support loading time since epoch from files (e.g. nexus) we need a way to convert ints to datetimes. And requiring to add it onto `sc.epoch()` would pessimise performance and double the memory cost.

Reply:
Ok, leave it for now.